core/src/repair_service.rs

```diff
@@ -595,6 +626,7 @@ impl RepairService {
                 slot,
                 &meta,
                 max_repairs - repairs.len(),
+                Duration::from_millis(200),
```

Comment:
Should we use `DEFER_REPAIR_THRESHOLD` here (and other places we're using `Duration::from_millis(200)`)?

Reply:
I'll change `Duration::from_millis(200)` to `DEFER_REPAIR_THRESHOLD` in repair tests.